### DISCUSSION TITLE
How to perform a task on the last day of the month?

### DISCUSSION THREAD

**Discussion by msisolucoes** [cybertec-postgresql/pg_timetable#660] [Q&A] [answered]
I used the cron expression '* * L * *' but it didn't work.
Is there any website online where I can set up my condition to set up the cron expression?
Waiting for your comments, thank you.

**Comment by pashagolub** [comment, marked as answer]
Hello.

`L` is not part of a [cron syntax](http://crontab.org/). 

I suggest you to run your job at `0 0 1 * *` which is 00:00 of 1st day of the month.
